### ISSUE TITLE
Failed to Create D3D Device

### ISSUE THREAD

**Issue by V01d-dev** [xoxor4d/p2-rtx#18]
I'm getting this error when I'm trying to start Portal 2 with RTX

**Comment by FluffyPuppyKasey** [comment]
No one can help you if you give us no information.

What hardware? What OS? What driver version?

**Comment by V01d-dev** [comment]
CPU - I5 9400f
GPU - RTX 4060Ti
RAM - 32Gb
Windows
Last driver version

**Comment by xoxor4d** [comment]
Does Portal (1) RTX work for you? Or any other game running with remix?

**Comment by V01d-dev** [comment]
Yes, Portal with RTX works on my setup

**Comment by Hansi137** [comment]
Same error here,

downloaded both .ZIP files, extracted and copied to the SteamApps Common Portal 2 root directory where the Portal 2.exe is located. Tried to start the .bat file but got the D3D Error

**Comment by V01d-dev** [comment]
I don't know why, but it started when I installed RTX Remix from Nvidia Omniverse

**Comment by OrsellGit** [comment]
Sometimes, this happens when the game tries to run with an unsupported resolution because sometimes it bugs out like that. Running the game with the `-safe` launch option flag should work as it loads the game at a low resolution. From there, you should be able to fix the resolution, close the game, and start it again with the rtx mod. Hopefully, then, it should work.

**Comment by rwolf2467** [comment]
I have the same error. I have tried the method, but unfortunately it still does not work.

**Comment by xoxor4d** [comment]
@rwolf2467 Mhh so even installing the RTX Remix Toolkit did not help? Did you restart your pc after installing it?
Do you have a secondary? If so, try unplugging it and try launching the game again

**Comment by rwolf2467** [comment]
Yes, I downloaded and installed RTX Remix via the NVIDIA app. Unfortunately, I get the same error.
